components/header.js

```diff
@@ -1,34 +1,72 @@
-import Link from "next/link"
+import Link from "next/link";
+import styles from "./style.module.css";
+import Image from "next/image";
+
+import React, { useEffect, useState } from "react";
 
 export default function Header() {
-    
-  
-    return (
-      <header>
-        <nav>
-          <ul className="">
-            <li className="">
+  const [items, setItems] = useState([]);
+
+  useEffect(() => {
+    fetch("http://turismo:8081/rest/weather")
+      .then((response) => response.json())
+      .then((json) => setItems(json))
+      .catch((error) => alert("Error" + error.message));
+  }, []);
+  const weather = {
+    main: items.main,
+    icon: items.icon,
+    celsius : items.celsius,
+  };
+
+  return (
+    <header>
+      <div className={styles.cabecera}>
+        <nav className={styles.navegator}>
+          <ul className={styles.listaNav}>
+            <li className={styles.liNav}>
               <Link href="/">
                 <a>Home</a>
               </Link>
             </li>
-            <li className="">
-              <Link href="/client">
+            <li className={styles.liNav}>
```

Comment:
why not have an array with [{href, name}] to render all the li at once?

Reply:
I changed this form and i created array and push into this my nav pages. After into return i put and if because i dont know how used redirect. look that and say me if i do of the correct form